mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -178,7 +178,7 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 4 y
 		ARG 5 color
 		ARG 6 rightToLeft
-	CLASS class_5232 ShadowDrawer
+	CLASS class_5232 Drawer
```

Comment:
This could be confused with the type of furniture. Maybe `Renderer`?

Reply:
Could work too, will wait a bit for opinions before renaming it

Reply:
Wouldn't it be weird to have a TextRenderer.Renderer class?

Reply:
Maybe `TextDrawer` to not have confusion from a plain `Drawer`?

Reply:
Who will confuse an inner class of TextRenderer with a furniture?

Reply:
Good question. :tiny_potato: